_data/projects/githunt.yml

```diff
@@ -0,0 +1,15 @@
+---
+name: githunt
+desc: Browse most stared Github repositories by date from your command line.
+site: https://github.com/SriNandan33/githunt
+tags:
+- python
+- click
+- colorama
+- cli
+upforgrabs:
+  name: up for grabs
+  link: https://github.com/SriNandan33/githunt/labels/up%20for%20grabs
+stats:
+  issue-count: 1
+  last-updated: '2020-01-18T13:53:58.907Z'
```

Comment:
@SriNandan33 You don't need to include this. it will be automatically added later.

Reply:
Hoo okay. Should I remove the key also `last-updated`? or just the timestamp value and keep the empty string (last_updated: '' )?

Reply:
remove all three lines, 13, 14, 15 @SriNandan33

Reply:
Okay, Thanks @ritwik12

Reply:
Removed lines 13,14,15 @ritwik12 .